File: Python/Ex36.py
numeros = []
equivalente = []
for i in range(0, 20, 1):
    x = int(input('Digite um numero: '))
    numeros.append(x) 

constante = int(input('Digite a constante multiplicativa: '))
for i in range(0, 20, 1):
    y = numeros[i]*constante 
    equivalente.append(y) 
 
print('Os numeros digitados multiplicados pela constante: ')

for i in range(0, 20, 1):
    print(equivalente[i])

File: C/Ex36.c
#include <stdio.h>
#include <stdlib.h>
int main(int argc, char *argv[]){
  int numeros[20];
  int multiplicador;
  int resultado[20];
  
  printf("Digite os 10 numeros: ");
  for(int i=0; i<20; i++){
      scanf("%i", &numeros[i]);
  printf("Deve multiplicar por quanto? \n");
  scanf("%i", &multiplicador);
    resultado[i] = numeros[i] * multiplicador;
  }
 
  printf("Os numeros digitados multiplicados pelos respectivos valores:  ");
  for(int i=0; i<20; i++){
      printf("%i \n", resultado[i]);
  }
  return 0;
}